### PR TITLE
Adds "Introducing Aurora" post

### LINKED ISSUE #5044: Introducing Aurora [5/10/2021]
* [Draft](https://docs.google.com/document/d/1chJSNnrZn3wRoNTaJa045FbIP9A-cY73QVlqceg6uvo/edit?usp=sharing&resourcekey=0-UJRoTSNwIy-EC9hTwaAJrg) - Google internal and not to be shared with non-Googlers until the post is published.

### PR DESCRIPTION
**Pull request by housseindjirdeh**
Fixes #5044

Reviewed here: https://docs.google.com/document/d/1chJSNnrZn3wRoNTaJa045FbIP9A-cY73QVlqceg6uvo/edit?usp=sharing&resourcekey=0-UJRoTSNwIy-EC9hTwaAJrg

### CODE REVIEW

File: src/site/content/en/blog/introducing-aurora/index.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+An aurora is a natural light display that shimmers in the sky. As we are trying
+to help user experiences built with frameworks shimmer and shine, we thought
+this name was an appropriate choice.
+
+{% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/lxaUR7Za1DoQMJuJEZHP.png",
+alt="Aurora logo", width="800", height="450" %}
+
+In the coming months, we'll be sharing a lot more detail on Aurora. This is a
+collaboration between a small team of Chrome engineers (internally codenamed
+WebSDK) with experience working on front-end stacks, and framework and tooling
```
Comment (spanicker) [inline src/site/content/en/blog/introducing-aurora/index.md:53]: this doesn't read right. What about:
"... , and contributing to frameworks and tooling"

File: src/site/content/en/blog/introducing-aurora/index.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+Frameworks have a unique vantage point for influencing both DX and UX as they
+span the entire system: the client and the server, the development and
+production environments, and they integrate tools such as compiler, bundler,
+linter etc.
+
+<figure class="w-figure">
+  {% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/UuOwHlcV76N8BwboHBVy.png", alt="Chart that shows
+  common tooling in frameworks", width="800", height="450" %}
+  <figcaption class="w-figcaption">
```
Comment (spanicker) [inline src/site/content/en/blog/introducing-aurora/index.md:73]: left a doc comment on the diagram

File: src/site/content/en/blog/introducing-aurora/index.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+When solutions are baked into the framework, teams of developers can use these
+solutions and focus their time on what matters most to the product -- shipping
+great features for users.
+
+While we work to improve tools that live in every layer of the stack, frameworks
+such as Next.js, Nuxt.js, and Angular CLI, manage every step of an application's
+lifecycle. For this reason, and the fact that React adoption is the largest
+within the core UI framework ecosystem, most of our optimizations have begun as
+a proof of concept within Next.js before expanding to the rest of the ecosystem.
```
Comment (spanicker) [inline src/site/content/en/blog/introducing-aurora/index.md:86]: "begun with proving out in Next.js" 
(being more direct: we are shipping production features here and not just doing proof of concept work)

File: src/site/content/en/blog/introducing-aurora/index.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+{% endAside %}
+
+## What is our work process?
+
+Our principles for how Aurora bridges browsers and the developer ecosystem are:
+humility, curiosity, scientific enquiry and pragmatism. We work with framework
+authors on improvements, collaborate with the community and do our due diligence
+prior to landing any changes.
+
+{% Img src="image/0SXGYLkliuPQY3aSy3zWvdv7RqG2/oRxGw4dKoQJRhke09t86.png",
```
Comment (spanicker) [inline src/site/content/en/blog/introducing-aurora/index.md:105]: can you update the image?
screenshot of this:
https://docs.google.com/presentation/d/16n6ywPHIJm4wckZVLTsqswbD2LEYdPFj-frFX5Q1iSg/edit#slide=id.gd8b919ad07_0_142

File: src/site/content/en/blog/introducing-aurora/index.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+1. Identify user-experience pain in a popular stack, using representative apps.
+2. Prototype solutions that address this, with an emphasis on "strong defaults"
+   .
+3. Verify the feature with another framework stack, to ensure it is adaptable.
+4. Validate the feature by experimenting in a few production apps, typically with
+   lab testing for performance.
+5. Drive design using the RFC process, addressing community feedback.
+6. Land the feature in a popular stack, typically behind a flag.
+7. Enable the feature in a representative production app to assess quality,
+   metrics and developer workflow integration.
```
Comment (spanicker) [inline src/site/content/en/blog/introducing-aurora/index.md:120]: "to assess quality and developer workflow integration"
drop metrics, as that's the next line

File: src/site/content/en/blog/introducing-aurora/index.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+  in Next.js that includes a custom plugin and shareable configuration to make
+  it easier to catch common framework-specific issues at build-time, resulting
+  in more predictable loading performance.
+- An introduction of a built-in performance relayer in [Create React
+  App](https://create-react-app.dev/docs/measuring-performance/) and
+  [Next.js](https://nextjs.org/docs/advanced-features/measuring-performance) to
+  allow easier insight into page performance through web vitals and other custom
+  metrics.
+- [Granular chunking](/granular-chunking-nextjs/) shipped in
+  Next.js and Gatsby, resulting in reduced bundle sizes while improving caching
```
Comment (spanicker) [inline src/site/content/en/blog/introducing-aurora/index.md:181]: resulting in reduction bundle sizes by upto 70%, while improving caching

Comment (housseindjirdeh) [reply src/site/content/en/blog/introducing-aurora/index.md]: Updated

File: src/site/content/en/blog/introducing-aurora/index.md (hunk 5 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/introducing-aurora/index.md]: Updated

File: src/site/content/en/blog/introducing-aurora/index.md (hunk 4 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/introducing-aurora/index.md]: Updated

File: src/site/content/en/blog/introducing-aurora/index.md (hunk 3 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/introducing-aurora/index.md]: Updated

File: src/site/content/en/blog/introducing-aurora/index.md (hunk 2 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/introducing-aurora/index.md:72]: Updated per comment (added a label 👍 )

File: src/site/content/en/blog/introducing-aurora/index.md (hunk 1 above)
Comment (housseindjirdeh) [reply src/site/content/en/blog/introducing-aurora/index.md]: Yeah I think that still sounds a little confusing, so I simplified it to "between a small team of Chrome engineers and framework authors"

File: src/site/content/en/blog/introducing-aurora/index.md (hunk 6 above)
Comment (spanicker) [reply src/site/content/en/blog/introducing-aurora/index.md]: nit: 30% to 70% (to be more representative)

**Review by addyosmani — APPROVED** [review]
LGTM to land (coordinating with the team)

**Comment by samuelfernandez** [comment]
@spanicker This article is super cool. I’d be interested in helping to build something similar to enhance image optimization for Angular apps, most likely as an express middleware? Please let me know if you have any plans at this regard, and how to contribute from community to Aurora project

**Pull request merged**